### PR TITLE
fix: modify components p-z to buttonv2

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Modified components p-z to buttonV2

Attached screenshots of affected buttons:
Screenshot 2023-12-22 at 11 45 34
Screenshot 2023-12-22 at 11 47 01
Screenshot 2023-12-22 at 11 41 42
Screenshot 2023-12-22 at 11 33 30
Screenshot 2023-12-22 at 11 32 17
Screenshot 2023-12-22 at 11 55 15
Screenshot 2023-12-22 at 11 56 41
Screenshot 2023-12-22 at 11 59 41
Screenshot 2023-12-22 at 12 05 37
Screenshot 2023-12-22 at 12 09 54
Screenshot 2023-12-22 at 11 53 06
Screenshot 2023-12-22 at 11 49 42
Screenshot 2023-12-22 at 12 20 50
Screenshot 2023-12-22 at 12 22 46
Screenshot 2023-12-22 at 12 25 54

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/search/SearchResult.tsx [hunk 1]
```diff
@@ -125,24 +125,24 @@ export function SearchResult({
...
+            className="mr-2"
             pressed={chunk.feedback === 1}
             icon={<UpvoteIcon secondary={chunk.feedback === 1} />}
             onClick={() => sendFeedback(chunk.feedback === 1 ? 0 : 1)}
             disabled={isInProgress}
           />
           <Button
-            className="mr-2 btn-tertiary-ketchup"
-            iconOnly
+            variant={ButtonVariant.Tertiary}
```
Comment (kopancek) [inline packages/shared/src/components/search/SearchResult.tsx:137]: we're missing the  `mr-2` className now, is it not needed?

File: packages/shared/src/components/tooltips/InteractivePopup.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ function InteractivePopup({
   position = InteractivePopupPosition.Center,
   closeOutsideClick,
   onClose,
-  closeButtonTypeClassname = 'btn-secondary',
+  closeButtonTypeClassname = ButtonVariant.Secondary,
```
Comment (kopancek) [inline packages/shared/src/components/tooltips/InteractivePopup.tsx:69]: Should we change the name of the parameter as well? E.g. `closeButtonVariant` instead.

**Review by kopancek — CHANGES_REQUESTED** [review]
Looks great and so fast as well 😮 

Left some small comments.

File: packages/shared/src/components/tooltips/InteractivePopup.tsx (hunk 2 above)
Comment (rebelchris) [reply packages/shared/src/components/tooltips/InteractivePopup.tsx:69]: We can do :) that's fine with me.
I'll adapt it.

**Review by kopancek — APPROVED** [review]
🚢

**Pull request merged**